recipes-nymea/nymea-zigbee/files/0001-Make-qca-optional-only-required-for-TI-backend.patch

```diff
@@ -0,0 +1,55 @@
+From 932a4bcd22a529c9437f1ea12a445520ab75898b Mon Sep 17 00:00:00 2001
+From: Lukas Pfeifhofer <lukas.pfeifhofer@streamunlimited.com>
+Date: Mon, 7 Feb 2022 10:51:34 +0100
+Subject: [PATCH 1/1] Make qca optional, only required for TI backend.
+
```

Comment:
You could send this patch to nymea-zigbee git repo so that the layer does not need to carry this patch for too long (next upgrade it'd be gone).

Reply:
Yes planning to send this upstream.

Reply:
Sent: https://github.com/nymea/nymea-zigbee/pull/51